### DISCUSSION TITLE
How to set thumbnail view

### DISCUSSION THREAD

**Discussion by SeraphofNineball** [aleksey-hoffman/sigma-file-manager#210] [Q&A]
Is there a way to show image thumbnails without using the preview pane?

Also how do you increase the size of the image in the preview pane?

**Comment by aleksey-hoffman** [comment]
1. Is it not showing any thumbnail previews for images on the item cards for you?

![image](https://user-images.githubusercontent.com/61761672/219981197-f1c5abb6-1bc8-4412-8d0d-6d7290c7ac23.png)

2. Right now you cannot change the info panel width. I will add this ability in the future

**Reply by SeraphofNineball** [reply]
Correct, This is what i see. I also tried the list layout view and nothing as well.
![Screenshot 2023-02-19 181555](https://user-images.githubusercontent.com/82607191/219981237-fdedb14e-2c22-4a7b-84e1-c58e175c2d3c.png)

**Reply by aleksey-hoffman** [reply]
Can you pleaase open the app console by pressing `Ctrl + Shift + I` and screenshot the "Console" tab? Do you see any red errors, or text mentioning "ffmpeg" or sometihng like that?

![image](https://user-images.githubusercontent.com/61761672/219981463-4f68e2cf-7651-4853-987b-474326d51be3.png)

**Reply by SeraphofNineball** [reply]
I can't get the console to open with that key combination. I'm running the windows store app version, if that impacts anything.

**Reply by aleksey-hoffman** [reply]
The app should be focused when you press it

Also, make sure you are pressing the "i" button (to the right of "u" button on your keyboard), not the "L"

**Reply by SeraphofNineball** [reply]
Yes.... Come on man, im an systems engineer :) . I pressed  Ctrl + Shift + I  and nothing happens, the dev console doesn't open.

**Reply by aleksey-hoffman** [reply]
Good to know you're an engineer and can help me figure it out :)

Would you try installing the provided exe file instead of the MS store version and see if that works?

**Reply by SeraphofNineball** [reply]
Alright so i installed the executable version instead and now thumbnails are loading but the keyboard combo still doesn't work for opening the dev console.

**Reply by aleksey-hoffman** [reply]
That's strange, I've never encountered this issue. 
The closest thing I remember is when it was opening on another monitor and I couldn't see it :D, but I doubt it's the same case here

**Reply by GieriGuru** [reply]
have the same behavior. 🤷‍♂️
No Thumbnails to see. Will try the exe .

BTW. is it possible to group the files as well? So group first for file type and then for latest access

**Reply by aleksey-hoffman** [reply]
> BTW. is it possible to group the files as well? So group first for file type and then for latest access

Not yet. But I'm working on this feature in the v2.0 version

**Comment by GieriGuru** [comment]
Nice 😍

If you need beta tester, count me in?

BTW. Thumbview works in the .exe version 👍

Cheers
Markus

----------------------------------------

08.06.2023 14:14:54 Aleksey Hoffman ***@***.***>:

**Reply by aleksey-hoffman** [reply]
> If you need beta tester, count me in?

Thanks for asking. I do need a beta tester :)
I just uploaded a new feature and will post a video preview later today on [Twitter ](https://twitter.com/sigma__dev) and [reddit/r/SigmaFileManager](https://www.reddit.com/r/SigmaFileManager). I will create a "beta testing" thread here and perhaps on Reddit and send the pre-release exe file there. 

I will add a few more features in today and tomorrow as well

**Reply by aleksey-hoffman** [reply]
@GieriGuru the v1.7 update was released! If you'd like to test it out, you can see the list of changes and download it here:
https://github.com/aleksey-hoffman/sigma-file-manager/releases

**Comment by GieriGuru** [comment]
Hi Aleksey,

thx for the new version.
I guess the exclusion filter in the global search options are not 
working for me.

I exclude some drives, but the search still indexing them. What could be 
the problem here?
I still generating search_data_X.txt files. Each for each drive which I 
have excluded.

Cheers
Markus



------ Originalnachricht ------
Von "Aleksey Hoffman" ***@***.***>
An "aleksey-hoffman/sigma-file-manager" 
***@***.***>
Cc "GieriGuru" ***@***.***>; "Mention" 
***@***.***>
Datum 11.06.2023 22:11:19
Betreff Re: [aleksey-hoffman/sigma-file-manager] How to set thumbnail 
view (Discussion #210)

**Reply by aleksey-hoffman** [reply]
> I exclude some drives, but the search still indexing them.

Right now you can indeed only hide the drives from search results, not to exclude them from indexing. I will add an option to exclude drives later.
